clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -67,6 +68,140 @@ trait GvcfTests { self: BaseIntegrationSpec =>
     it should behave like testGvcfLocation(_)
   }
 
+  it should "handle raw queries for specific fields only" in {
+    val sampleAlias = s"someAlias $randomId"
+    val key = GvcfKey(
+      location = Location.GCP,
+      project = "test project",
+      sampleAlias = sampleAlias,
+      version = 2
+    )
+    val gvcfPath = Some(URI.create(s"gs://path/gvcf${GvcfExtensions.GvcfExtension}"))
+    val metadata = GvcfMetadata(
+      gvcfPath = gvcfPath,
+      documentStatus = Some(DocumentStatus.Normal)
+    )
+    val expected = Map("gvcf_path" -> gvcfPath).asJson
+    val rawJsonQuery =
+      s"""{
+         |  "_source": "gvcf_path",
+         |  "query": {
+         |    "bool": {
+         |      "must": [
+         |        {
+         |          "query_string": {
+         |            "default_field": "sample_alias.exact",
+         |            "query": "\\"$sampleAlias\\""
+         |          }
+         |        }
+         |      ]
+         |    }
+         |  }
+         |}
+      """.stripMargin
+    rawQueryTest(
+      rawJsonQuery,
+      runUpsertGvcf(key, metadata),
+      outputs => {
+        outputs should have length 1
+        outputs.headOption should be(Some(expected))
```

Comment:
I think these two could be collapsed into `outputs should contain only expected`